poly-commit/src/lib.rs

```diff
@@ -665,12 +654,9 @@ pub mod tests {
         PC: PolynomialCommitment<F, P, S>,
         S: CryptographicSponge,
     {
-        let challenge_generators = vec![
-            ChallengeGenerator::new_multivariate(sponge()),
-            ChallengeGenerator::new_univariate(&mut sponge()),
-        ];
+        let sponge = sponge();
 
-        for challenge_gen in challenge_generators {
+        for __ in 0..1 {
```

Comment:
By deleting this for loop, the diff gets unnecessarily complicated. As soon as the PR gets a review, I will delete this for loop, and then please merge it.

Reply:
PR is approved!